src/main.d

```diff
@@ -169,7 +169,7 @@ else
 
 	immutable usingStdin = args.length == 1;
 
-	StringCache cache = StringCache(StringCache.defaultBucketCount);
+	StringCache cache = StringCache(0x10000);
```

Comment:
Whatever this number will be, imho it should definitely be accompanied by an explanatory comment ;-)